File: node/src/Container.js
module.exports = class Container {

  constructor() {
    this.instances = {}
  }

  get(name) {
    return this.instances[name]
  }

  has(name) {
    return this.instances[name] !== undefined
  }

  set(name, value) {
    this.instances[name] = value
  }
}

File: python/src/Container.py
class Container:
  instances = {}
  
  def set(self, name, instance):
    self.instances[name] = instance
  
  def get(self, name):
    if name not in self.instances:
      return None
    return self.instances[name]

  def count(self):
    return len(self.instances)

  def getInstances(self):
    return self.instances
